.github/workflows/pull_request.yml

```diff
@@ -179,6 +179,17 @@ jobs:
           # shellcheck disable=SC2046
           docker rm -f $(docker ps -a -q) ||:
           sudo rm -fr "$TEMP_PATH" "$CACHES_PATH"
+  SimpleCheck:
```

Comment:
I don't get it. The plan is to create a commit status, like our other statuses, see the screenshot 
![image](https://user-images.githubusercontent.com/3025537/177113416-0ff7048d-1daa-43f5-9eb5-c15f361c09c2.png)

You'll slip into hell with solutions based on GH actions. Better add a status to something global like https://github.com/ClickHouse/ClickHouse/blob/master/tests/ci/run_check.py, and then to the checks we want blocking merge, like https://github.com/ClickHouse/ClickHouse/blob/master/tests/ci/fast_test_check.py and https://github.com/ClickHouse/ClickHouse/blob/master/tests/ci/build_check.py adding overwriting for the same status. See examples for https://github.com/ClickHouse/ClickHouse/blob/master/tests/ci/fast_test_check.py#L17